Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-FireEye_Red_Team_Tools_Investigation_and_Response.yml

```diff
@@ -0,0 +1,1242 @@
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "21"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 2580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "20":
+    id: "20"
+    taskid: dd67ba41-5a3d-48f9-8039-7c8c60386478
+    type: regular
+    task:
+      id: dd67ba41-5a3d-48f9-8039-7c8c60386478
+      version: -1
+      name: Manually isolate endpoints
+      description: Manually Isolate the compromised hosts.
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "19"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 1130,
+          "y": 2410
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "21":
+    id: "21"
+    taskid: 1b3be322-7b30-4ce5-8e7c-3c87eefe170f
+    type: regular
+    task:
+      id: 1b3be322-7b30-4ce5-8e7c-3c87eefe170f
+      version: -1
+      name: Investigate further
+      description: Further investigate the incident (endpoints, the entry point, etc’).
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "22"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 2720
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "22":
+    id: "22"
+    taskid: 1f6548cc-a780-4119-8847-03f212e7b3f3
+    type: regular
+    task:
+      id: 1f6548cc-a780-4119-8847-03f212e7b3f3
+      version: -1
+      name: Close investigation
+      description: commands.local.cmd.close.inv
+      script: Builtin|||closeInvestigation
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "23"
+    scriptarguments:
+      assetid: {}
+      closeNotes: {}
+      closeReason: {}
+      emailclassification: {}
+      id: {}
+      phishingsubtype: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 2890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "23":
+    id: "23"
+    taskid: c72f7b03-9d64-4898-821c-6ec51309f5b6
+    type: title
+    task:
+      id: c72f7b03-9d64-4898-821c-6ec51309f5b6
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 920,
+          "y": 3060
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "24":
+    id: "24"
```

Comment:
Consider switching to findIncidentsWithIndicator
Provides a much neater output and context and without the bug with the strange output (worth opening a separate issue for it).